tests/containers/podman_bci_systemd.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman
+# Summary: Test podman with systemd
+# Maintainer: qa-c@suse.de
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use utils qw(script_retry);
+use containers::utils qw(check_min_runtime_version);
+use serial_terminal 'select_serial_terminal';
+use version_utils qw(is_sle is_opensuse is_staging);
+use containers::k8s qw(install_k3s uninstall_k3s);
+use Utils::Architectures qw(is_ppc64le);
+
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $podman = $self->containers_factory('podman');
+    $self->{podman} = $podman;
+
+    my $image = get_var("CONTAINER_IMAGE_TO_TEST", "registry.suse.com/bci/bci-init:latest");
+
+    record_info('Test', 'Launch a container with systemd');
+    assert_script_run("podman run -d -p 80:80 --name nginx $image");
+
+    record_info('Test', 'Install nginx');
+    assert_script_run("podman exec -t nginx zypper -n in nginx");
+
+    record_info('Test', 'Start nginx');
+    assert_script_run("podman exec -t nginx systemctl start nginx");
+
+    record_info('Nginx service status', script_output("podman exec -t nginx systemctl status nginx"));
+
+    record_info('Test', 'Curl localhost');
+    assert_script_run("podman exec -t nginx curl http://localhost");
```

Comment:
This throws a 403 and `curl` is still happy. Let's also create a predefined page, which we can validate

```suggestion
    assert_script_run("podman exec -t nginx bash -c 'echo testpage123-content > /srv/www/htdocs/index.html'
    validate_script_output("podman exec -t nginx curl -sfL http://localhost", qr/testpage123-content/ );
```

Reply:
done